src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { getMovie } from '../../api';
+// import { MovieData } from '../../types/MovieData';
+// import { ResponseError } from '../../types/ReponseError';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+// const defaultMovie = {
+//   title: '',
+//   description: '',
+//   imgUrl: '',
+//   imdbUrl: '',
+//   imdbId: '',
+// };
+
+interface Props {
+  addMovie: (movie: Movie) => void;
+}
+
+const defaultImage
+  = 'https://via.placeholder.com/360x270.png?text=no%20preview';
```

Comment:
```suggestion
const defaultImage = 'https://via.placeholder.com/360x270.png?text=no%20preview';
```
it this case its better to disable eslint for one line